### ISSUE TITLE
[Discussion] Calculating tax for "interest payments"

### ISSUE THREAD

**Issue by pi0neerpat** [rtoken-project/rtoken-analytics#4]
## Problem

Sending "interest" to a 3rd party is probably a taxable-event. But there are outstanding questions which need to be addressed. Eventually the answers to these questions will be used to create additional features here, e.g., generating tax-receipts for charity donors.

1. Is sending the interest to someone considered "earned income" for the sender, the recipient, or both? There's a lot to unpack here, so maybe we should start with which scenario we want to focus on first- charities, product payments, investors, etc.
2. Calculating the precise amount of interest that Kate sends to Bob is difficult, due to the nature of how Compound interest is earned (e.g., block time). What's the best way to calculate the interest earned, and how precise do we actually need to be

## Open discussion

We welcome anyone who has thoughts on these topics!

**Comment by hellwolf** [comment]
(1) As far as I understand recipient definitely HAS to account for the interest earned, at the same time sender may use it for deductions if recipient qualifies in certain categories such as non-profit, charities, religious orgs, etc. Although highly depending on the jurisdiction, it seems quite plausible use cases.

(2) yes, we can only know the precise amount denominated in savings asset unit, and we will need some sort of formula approved by IRS that converts those values into USD... :D

**Comment by pi0neerpat** [comment]
I don't think we need approval from IRS. We just need to agree on whats the most accurate and "good-faith" method (for both sender and recipient). Aside from taxes, we are still incentivized to find an accurate estimate, since this allows businesses / content producers to get a better estimate of their income. And spenders to see more accurate spending. I guess I've convinced myself that as long as we do our best we will be ok.
